backend/Watchdog.Core/Watchdog.Core.API/Controllers/UsersController.cs

```diff
@@ -44,5 +44,12 @@ public async Task<ActionResult<UserDto>> UpdateAsync(int userId, UpdateUserDto u
             var user = await _userService.UpdateUserAsync(userId, updateUserDto);
             return Ok(user);
         }
+
+        [HttpGet("organization/{orgId}/notInOrg/")]
+        public async Task<ActionResult<IEnumerable<UserDto>>> GetMembersNotInOrganization(int orgId, string memberEmail = "")
```

Comment:
It would be better to set a default value in a service

Reply:
That was done for adding a query param if we don't need all the members in organization, example : organizaiton/1/notInOrg/ -> All members not in organization
organization/1/notInOrg/?memberEmail=@yahoo -> Members whocontain  '@yahoo' string in email

Reply:
This default parameter works as query param, it's just for controller. 
The query looks like organizqation/1/notInOrg/?memberEmail=@yahoo -> members who not in organization but email contains '@yahoo'
Or organization/1/notInOrg/ -> All members not in organization